templates/rails/Gemfile.lock

```diff
@@ -0,0 +1,268 @@
+GEM
```

Comment:
is Gemfile.lock something good to add since this file is going to be regenerated everytime you run bundle